app/controller.py

```diff
@@ -33,7 +33,7 @@ def get_article(article):
 
 @api.route('/v1/articles/<article>/not-interested', methods=['POST'])
 def article_not_interested(article):
-    set_article_not_interested(request.json, article)
+    set_article_not_interested(article, request.json)
```

Comment:
Kdyz je problem v poradi argumentu, mozna by stalo za to pouzit keyword argumenty, tj.:
```
set_article_not_interested(
    article_id=article,
    user_data=request.json,
)
```

Reply:
Fixed